playbooks/roles/ocp-customization/defaults/main.yaml

```diff
@@ -4,4 +4,5 @@
 workdir: ~/ocp4-workdir
 rhcos_kernel_options: []
 sysctl_tuned_options: false
+powervm_rmc: true
```

Comment:
The RMC support is useful for PowerVM, not for KVM. So the `powervm-rmc` should be true for PowerVM, false for KVM.